src/bin/unit_test_attribute.mk

```diff
@@ -1,5 +1,5 @@
 TARGET		:= unit_test_attribute
 SOURCES		:= unit_test_attribute.c
 
-TGT_PREREQS	:= $(LIBFREERADIUS_SERVER) libfreeradius-tacacs.a libfreeradius-util.a
+TGT_PREREQS	:= $(LIBFREERADIUS_SERVER) libfreeradius-util.a
```

Comment:
It does unless you've added test points

Reply:
https://github.com/FreeRADIUS/freeradius-server/pull/3567/commits/618bd09fe962a2758d1efaec6936720cd6f3a00b